packages/fetchai/skills/generic_buyer/tests/test_handlers.py

```diff
@@ -1017,7 +1015,7 @@ def test_handle_error(self):
             LedgerApiDialogue,
             self.prepare_skill_dialogue(
                 dialogues=self.ledger_api_dialogues,
-                messages=self.list_of_ledger_api_messages[:4],
+                messages=self.list_of_ledger_api_messages[:1],
```

Comment:
same question as in PR #496. We now only use indices slice(0, 2) and [4], but not [3] and [5]. Please explain.

Reply:
@Karrenbelt 
https://github.com/valory-xyz/open-aea/pull/496#discussion_r1056065249